CIPs/0019.md

```diff
@@ -0,0 +1,72 @@
+---
+cip: 0019
```

Comment:
As per my other point, the CIP number will be changed as only editors can specify one. I'll get back to you on the name later as it's not the most urgent part.